velox/functions/lib/window/Ntile.cpp

```diff
@@ -115,7 +117,7 @@ class NtileFunction : public exec::WindowFunction {
         vector_size_t numRows,
```

Comment:
Should above computeBucketValue function https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/window/Ntile.cpp#L104 be changed to return TResult type as well (instead of always int64_t) ?